R/map.R

```diff
@@ -145,9 +150,9 @@ each3 <- function(.x, .y, .z, .f, ...) {
 #' @rdname map2
 each_n <- function(.l, .f, ...) {
   args_list <- recycle_args(.l) %>% zip()
-  for (args in args_list) {
+  lapply(args_list, function(args) {
```

Comment:
oh ok, I didn't realize this loop came from the old code, I thought I introduced him mistakenly